test-cases/longevity/longevity-counters-multidc.yaml

```diff
@@ -18,6 +18,3 @@ nemesis_class_name: 'ChaosMonkey'
 nemesis_interval: 15
 
 use_mgmt: true
-
-server_encrypt: true
```

Comment:
leave server_encrypt in place, remove only client.

Reply:
done